package.json

```diff
@@ -29,15 +29,18 @@
     "check-typescript": "tsc",
     "check-type-coverage": "type-coverage --strict --ignoreCatch --atLeast=100",
     "check-all": "yarn check-prettier && yarn check-eslint && yarn check-typescript",
-    "postinstall": "husky install"
+    "postinstall": "husky install",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.12.0",
     "@hot-loader/react-dom": "17.0.1",
     "@react-pdf/renderer": "2.0.0",
     "@reduxjs/toolkit": "1.5.1",
+    "@tailwindcss/postcss7-compat": "^2.1.2",
```

Comment:
In this project, we use exact version of packages. Please remove `^` from this PR to follow the same pattern